code/modules/research/nodes/biotech.dm

```diff
@@ -1,6 +1,6 @@
 /datum/technology/basic_biotech
 	name = "Basic Biotech"
-	desc = "Basic biotech micro manipulations and scanning method."
+	desc = "Allows basic matter scanning and manipulation."
```

Comment:
We need to describe not to for what it need, we describe core data of technologies, for "Add this designs" or "Allow to use this" we have unlocks designs list